litmus-portal/frontend/src/components/UserManagementTab/CreateUser/index.tsx

```diff
@@ -0,0 +1,234 @@
+      });
+    } else {
+      setCreatePassword({
+        ...createPAssword,
+        err: true,
+        [prop]: event.target.value,
+      });
+    }
+  };
+
+  const handleClickShowPassword = () => {
+    setCreatePassword({
+      ...createPAssword,
+      showPassword: !createPAssword.showPassword,
+    });
+  };
+
+  const handleMouseDownPassword = (
+    event: React.MouseEvent<HTMLButtonElement>
+  ) => {
+    event.preventDefault();
+  };
+
+  // for personal details fields
+  const [email, setEmail] = React.useState<string>('');
+  const [userName, setUserName] = React.useState<string>('');
+  const [fullName, setFullName] = React.useState<string>('');
+
+  return (
+    <div>
+      <div className={classes.container}>
+        <div>
+          <div className={classes.suSegments}>
+            {/* Personal Details */}
+            <UserDetails
+              nameValue={fullName}
+              usernameIsDisabled={false}
+              handleNameChange={(e) => {
+                setFullName(e.target.value);
+              }}
+              emailValue={email}
+              handleEmailChange={(e) => {
+                setEmail(e.target.value);
+              }}
+              userValue={userName}
+              handleUserChange={(e) => {
+                setUserName(e.target.value);
+              }}
+            />
+
+            <Divider className={classes.divider} />
+
+            {/* Login Details */}
+            {showDiv ? (
+              /* Displays the reset password div */
+              <div>
+                <Typography className={classes.headerText}>
+                  <strong> Reset Password</strong>
+                </Typography>
+                <ResetModal
+                  resetPossible={fullName.length > 0 && userName.length > 0}
+                />
+                <div className={classes.copyDiv}>
+                  <img src={copy} alt="copy" />
+                  <Typography>Copy the credentials </Typography>
+                </div>
+                <DelUser handleModal={handleDiv} />
+              </div>
+            ) : (
+              /* displays create password div */
+              <div>
+                <Typography className={classes.headerText}>
+                  <strong> Login Details</strong>
+                </Typography>
+                <div>
+                  <form>
+                    <div className={classes.details1}>
+                      <TextField
+                        className={classes.userDetail}
+                        id="filled-username-input"
+                        label="Username"
+                        defaultValue="RichardHill"
+                        value={userName}
+                        disabled
+                        InputProps={{
+                          disableUnderline: true,
+                        }}
+                        data-cy="username"
+                      />
+                      <FormControl>
+                        <Input
```

Comment:
You can use InputField component here instead

Reply:
Yes, I was talking to @arkajyotiMukherjee regarding that, yesterday, he will be making some modifications in the InputField component, after I will change it everywhere at once.